### DISCUSSION TITLE
Clicking on a line number blanks the page

### DISCUSSION THREAD

**Discussion by jreut** [community/community#10078] [Other Feature Feedback, Questions, & Ideas] [answered]
Code search is great! But I think the version deployed as of 2022-01-13T15:00Z is broken.

When I am browsing a file, and I click on a line number, I expect the line to be highlighted and my browser's location hash to be updated. Instead, I get a blank screen and a classic JavaScript error.

https://user-images.githubusercontent.com/3959624/149355143-c812c279-6877-45ba-8897-df075786f6e5.mov

This happens on any repo I've tried, both public and private. I used rails/rails as an example in the recording. I can reproduce by navigating to `https://cs.github.com/rails/rails/blob/18e3226c9bf7265ce9e1edc41dee7581856fb574/version.rb` and clicking on any line number in the left-hand gutter. This is the text of the JavaScript error:

```
2.c4a67340.chunk.js:2 Uncaught TypeError: Cannot read properties of null (reading 'length')
    at main.fd9b22ce.chunk.js:1:80493
    at Mc (2.c4a67340.chunk.js:2:330716)
    at t.unstable_runWithPriority (2.c4a67340.chunk.js:2:348494)
    at qa (2.c4a67340.chunk.js:2:271035)
    at Nc (2.c4a67340.chunk.js:2:330177)
    at 2.c4a67340.chunk.js:2:330088
    at R (2.c4a67340.chunk.js:2:347553)
    at MessagePort.x.port1.onmessage (2.c4a67340.chunk.js:2:346275)
```

**Comment by colinwm** [comment, marked as answer]
@jreut thanks for reporting, sorry about that! It's fixed now
